src/node_buffer.cc

```diff
@@ -735,7 +735,9 @@ uint32_t WriteFloatGeneric(const FunctionCallbackInfo<Value>& args) {
 
   T val = args[1]->NumberValue();
   uint32_t offset = args[2]->Uint32Value();
-  CHECK_LE(offset + sizeof(T), ts_obj_length);
+  size_t memcpy_num = sizeof(T);
```

Comment:
ah I see. was focused on that `memcpy_num = sizeof(T)` was set then immediately after there's a check `sizeof(T)` in the conditional. nm it then. :)